src/style.css

```
@@ -0,0 +1,115 @@
+html {
+  font-size: 10px;
+  line-height: 14px;
+  color: #060b35;
+  font-family: Roboto, sans-serif;
+
+  --main-accent-color: #060b35;
+  --main-secondary-color: #616070;
+  --border-color: #f3f3f3;
+  --white-color: #fff;
+  --button-color: #00acdc;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+body {
+
+  margin: 0;
+}
+
+.card {
+  width: 200px;
+  padding: 32px 16px 16px;
+  background: var(--white-color);
+  border: 1px solid var(--border-color);
+  border-radius: 5px;
+}
+
+.card__image {
+  display: block;
+  margin: 0 auto 40px;
+  width: 160px;
+  height: 134px;
+}
+
+.card__title {
+  margin: 0 0 4px;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+  color: var(--main-accent-color);
+}
+
+.card__code {
+  margin-bottom: 16px;
+  color: var(--main-secondary-color);
+}
+
+.card__review-container {
+  display: flex;
+  justify-content: space-between;
+  align-items: flex-end;
+  margin-bottom: 24px;
+}
+
+.card__price-container {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin-bottom: 16px;
+}
+
+.card__price-label {
+  font-size: 12px;
+  line-height: 18px;
+  color: var(--main-secondary-color);
+}
+
+.card__price {
+  font-size: 16px;
+  line-height: 18px;
+  font-weight: bold;
```

it is better to use numeric value
```suggestion
  font-weight: 700;
```